### DISCUSSION TITLE
Ability to open the sprint on browser

### DISCUSSION THREAD

**Discussion by ricardovsilva** [ankitpokhrel/jira-cli#333] [Ideas] [answered]
My idea is to have something similar to a GitHub client that allows us to open a `repo,` `pr`, or `issue` directly on the browser.
In Github client, we can call something like:
```
gh pr view --web
```
And then, it opens the last PR directly on the browser.

So, with JIRA, we could do something like:
```
jira sprint ls --current --web
```

I can implement this feature by myself and submit the PR. I want to discuss its usability before implementing it.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @ricardovsilva, `jira open` will take you to the configured board, which is usually the active sprints page for scrum board. Isn't that what you are looking for?
